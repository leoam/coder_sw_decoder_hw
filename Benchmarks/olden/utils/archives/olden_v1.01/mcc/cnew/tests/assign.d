18: operands of = have illegal types `pointer to struct this' and `int'
.seg "text"

.global _main
ARGI(CNSTU(4))
CALLI(ADDRGP(mymalloc))
was node 0 of type call
with successors:1, 

ASGNP(ADDRLP(true), CVUP(CVIU(CALLI(ADDRGP(mymalloc)))))
CALLI(ADDRGP(MIGRATE))
was node 1 of type call
with successors:2, 

was node 2 of type other
with successors:3, 

CALLI(ADDRGP(MIGRPH))
was node 3 of type call
with successors:4, 

was node 4 of type other
with successors:5, 

ARGI(CNSTU(4))
CALLI(ADDRGP(mymalloc))
was node 5 of type call
with successors:6, 

ASGNP(ADDRLP(thing), CVUP(CVIU(CALLI(ADDRGP(mymalloc)))))
ASGNI(ADDRLP(y), INDIRI(INDIRP(ADDRLP(thing))))
ASGNP(ADDRLP(true), INDIRP(ADDRLP(thing)))
ASGNI(ADDRLP(y), INDIRI(INDIRP(ADDRLP(true))))
ASGNI(ADDRLP(y), INDIRI(INDIRP(ADDRLP(thing))))
ASGNI(ADDRLP(thing), INDIRI(ADDRLP(y)))
ASGNI(INDIRP(ADDRLP(thing)), INDIRI(ADDRLP(y)))
was node 6 of type other
with successors:7, 

ASGNI(ADDRLP(y), INDIRI(INDIRP(ADDRLP(true))))
was node 7 of type other
with successors:8, 

RETV()
was node 8 of type other
with successors:

LABELV(1)
was node 9 of type other
with successors:

