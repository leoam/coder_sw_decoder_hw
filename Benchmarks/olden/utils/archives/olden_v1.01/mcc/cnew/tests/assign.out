18: operands of = have illegal types `pointer to struct this' and `int'
.seg "text"

.global _main
-----------------------
************************
-----------------------
************************
-----------------------
true 
local:true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
true 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing 
true 
local:thing 
************************
-----------------------
thing true 
local:thing true 
************************
-----------------------
thing true 
local:thing true 
************************
-----------------------
thing true 
local:thing true 
************************
-----------------------
true 
local:true 
************************
-----------------------
true 
local:true 
************************
-----------------------
true 
local:true 
************************
-----------------------
true 
local:true 
************************
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:76ff0
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77050
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:770a0
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77100
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77150
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:771b0
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77200
all_work: 0
Nsucc: 1  howfar: 0
i:0 list:77250
all_work: 0
Nsucc: 0  howfar: 0
